js/components/discussions/thread-form.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+<form role="form" class="clearfix animated" @submit.prevent="submit">
+    <div class="form-group">
+        <label for="comment-new-message">{{ _('Comment') }}</label>
+        <textarea id="comment-new-message" v-model="comment" class="form-control" rows="3" required></textarea>
+    </div>
+    <button @click="submit" :disabled="this.sending || !this.comment" class="btn btn-primary btn-block pull-right submit-new-message">
+        {{ _('Submit your comment') }}
+    </button>
+</form>
+</template>
+
+<script>
+import i18n from 'i18n';
+import Auth from 'auth';
+import log from 'logger';
+
+
+export default {
+  props: {
+      discussionId: String
+  },
+  data() {
+      return {
+          sending: false
+      }
+  },
+  methods: {
+      submit(event) {
+          if (!Auth.need_user(i18n._('You need to be logged in to comment.'))) {
+              return;
+          }
+          this.sending = true;
+          this.$api
+          .post(`discussions/${this.discussionId}/`, {comment: this.comment}).then(response => {
+              this.$dispatch('discussion-load', response);
+              this.comment = '';
+          })
+          .catch(log.error.bind(log))
+          .chain(() => {
```

Comment:
I was looking for a `finally` and found this. I can't find the resource again.
I wish I could run a _debugger_ and inspect `this.$api` again but I can't run udata right now due to some branching breaking (auth/datagouv issues).
Proposals:
- postponing this PR until getting a proper udata again
- removing the `chain` method and duplicating the `this.sending = false` into the `then` and `catch` functions.